### ISSUE TITLE
Failed to load inline PDF using provided examples.

### ISSUE THREAD

**Issue by iapparatus** [enupal/snapshot#28]
### Description

Failed to load inline PDF using provided examples even though the PDF Download URL example is working.

### Steps to reproduce

1. Use the following example:

```
{% set html %}
	<!DOCTYPE html>
	<html lang="en">
	<head>
		<meta charset="UTF-8">
		<title>First Test</title>
	</head>
	<body>
		<h1>Hello world!</h1>
	</body>
	</html>
{% endset %}

{% set settings = {filename: 'my-first.pdf'} %}

{{craft.enupalsnapshot.displayHtml(html, settings)}}

```

See attached output.

### Additional info

- Craft version: 3.1.30
- PHP version: 7.2.5
- Database driver & version: MySQL 5.5.5
- Plugin version: 1.2.4

[10c5c61a07222bf9759e37236f85867b860be0421f4f8071f2da136a95db340d.txt](https://github.com/enupal/snapshot/files/3370102/10c5c61a07222bf9759e37236f85867b860be0421f4f8071f2da136a95db340d.txt)

**Comment by andrelopez** [comment]
Hi @iapparatus 

Could you please recreate the error and send back to us (support@enupal.com) a zip file of the following folder?

storage/logs

Are you able to recreate this issue in a different browser?

**Comment by andrelopez** [comment]
Hi @iapparatus 
We just released Enupal Snapshot v1.2.6 with a fix to this issue, could you please update the plugin and let us know if the issue is fixed?

Best

**Comment by iapparatus** [comment]
Fixed! Thank you!
